src/dmd/dsymbolsem.d

```diff
@@ -2831,6 +2831,67 @@ private extern(C++) final class DsymbolSemanticVisitor : Visitor
         if (!sc)
             return;
 
+        bool repopulateMembers = false;
+        if (ns.identExp)
+        {
+            // resolve the namespace identifier
+            sc = sc.startCTFE();
+            Expression resolved = ns.identExp.expressionSemantic(sc);
+            resolved = resolved.expressionSemantic(sc);
```

Comment:
Is it really necessary to call expressionSemantic again here?

Reply:
Oh, good catch! This must be a cut&paste fail.